ted_sws/mapping_suite_processor/entrypoints/cli/cmd_mapping_suite_processor.py

```diff
@@ -50,68 +86,154 @@ def __init__(
             raise FileNotFoundError(error_msg)
 
     def _cmd(self, cmd: str):
-        if cmd == 'normalisation_resource_generator':
-            cmd_generate_mapping_resources.run(
+        if cmd == 'resources_injector':
```

Comment:
replace string with a constant

Reply:
constants are good if there are more usages of the same string.
it's the case here